diesel/Cargo.toml

```diff
@@ -13,6 +13,7 @@ categories = ["database"]
 
 [dependencies]
 byteorder = "1.0"
+diesel_derives = {version = "0.16.0", path = "../diesel_derives/" }
```

Comment:
Remember to remove the `path` attr when this is released!

Also: Can you add a space after the opening brace to make it consistent with the other lines?